meson.build

```diff
@@ -0,0 +1,49 @@
+project('hack-components', version: '0.0.0')
+
+conf = configuration_data()
+
+cmd = run_command('sh', '-c', 'echo $FLATPAK_ARCH')
+conf.set('FLATPAK_ARCH', cmd.stdout().strip())
+
+cmd = run_command('sh', '-c', 'echo $BRANCH')
+conf.set('BRANCH', cmd.stdout().strip())
+
+backgrounds_install_dir = join_paths(get_option('datadir'), 'backgrounds')
+conf.set('BACKGROUNDSDIR', join_paths(
+    '/var/lib/flatpak/app/com.endlessm.HackComponents',
+    conf.get('FLATPAK_ARCH'),
+    conf.get('BRANCH'),
+    'active/files',
+    backgrounds_install_dir
```

Comment:
Are you sure that this does not include the prefix? Otherwise we'll set a path that does not exist.

Reply:
I have tested it:

```
$ tree build
build
├── export
├── files
│   ├── clippy
│   ├── hack-avatar-faces
│   ├── manifest.json
│   └── share
│       ├── backgrounds
│       │   ├── Desktop-BGs-Beta-Binary.png
│       │   ├── Desktop-BGs-Beta-Cat_Grid.png
│       │   ├── Desktop-BGs-Beta-Pop_Art.png
│       │   ├── Desktop-BGs-Beta-Sketch_Blue.png
│       │   ├── Desktop-BGs-Beta-Sketch_Green.png
│       │   ├── hack_colors_05_blurred-FBE.png
│       │   ├── hack_colors_05.png
│       │   ├── riley_colors_02b_blurred-FBE.png
│       │   ├── riley_colors_02b.png
│       │   └── saniel_colors_02b.png
│       ├── flatpak-overrides
│       │   └── global
│       ├── gnome-background-properties
│       │   └── hack-backgrounds.xml
│       └── hack-components
│           └── hack-data.ini
├── metadata
└── var
    ├── lib
    ├── run -> /run
    └── tmp
```

```
$ head build/files/share/gnome-background-properties/hack-backgrounds.xml 
<?xml version="1.0"?>
<!DOCTYPE wallpapers SYSTEM "gnome-wp-list.dtd">
<wallpapers>
  <wallpaper deleted="false">
    <name>Desktop-BGs-Beta-Binary.png</name>
    <filename>/var/lib/flatpak/app/com.endlessm.HackComponents/x86_64/master/active/files/share/backgrounds/Desktop-BGs-Beta-Binary.png</filename>
    <options>zoom</options>
    <shade_type>solid</shade_type>
    <pcolor>#3465a4</pcolor>
    <scolor>#000000</scolor>
```